pkg/print/list/base.go

```diff
@@ -172,16 +200,36 @@ func (b *BaseListPrinter) Print(ch <-chan event.Event, previewStrategy common.Dr
 			if err := formatter.FormatDeleteEvent(e.DeleteEvent); err != nil {
 				return err
 			}
+		case event.WaitType:
```

Comment:
TODO: This needs to be update for the new WaitEvent Operations. -Error +Pending +Timeout.